### ISSUE TITLE
Polaris Helm Chart 4.0.5 bug with ingress controller

### ISSUE THREAD

**Issue by tom-dalton-fanduel** [FairwindsOps/polaris#598]
Raising this here as I can't seem to find the repo for the helm chart? Is it open source/public? If not, what is the correct process for raising issues like this? I've struggled to see what changes exist in the helm chart between 4.0.4 and 4.0.5, and had to resort to downloading both and manually diffing them to work out why this behaviour has changed, which obviously isn't ideal!

### Installation Process

Trying to install polaris on an EKS 1.21 cluster.
https://artifacthub.io/packages/helm/fairwinds-stable/polaris

### Polaris Version

Using helm chart 4.0.5 or 4.0.6, I think this is polaris 4.0.

### Expected Behavior

Polaris is successfully installed in the cluster.

### Actual Behavior

```
...
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "serviceName" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "servicePort" in io.k8s.api.networking.v1.IngressBackend]
...
```

### Steps to Reproduce

Install polaris via the helm chart version 4.0.5 or 4.0.6 in an EKS 1.21 cluster.

### Additional Context

In the helm chart 4.0.4 , in `ingress.yaml`, we have

```
{{ $serviceName := printf "%s-dashboard" (include "polaris.fullname" .) -}}
apiVersion: networking.k8s.io/v1beta1
kind: Ingress
```

and then there's the ingress setup further on in the file

```
      - backend:
          serviceName: {{ $serviceName }}
          servicePort: 80
```

This all works fine.

In the helm chart 4.0.4 , in `ingress.yaml`, we now have

```
{{ $serviceName := printf "%s-dashboard" (include "polaris.fullname" .) -}}
{{- if not (.Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress") }}
apiVersion: networking.k8s.io/v1beta1
{{- else }}
apiVersion: networking.k8s.io/v1
{{- end }}
kind: Ingress
```

which seems to be a way to avoid using the deprecated `networking.k8s.io/v1beta1` API if new new (`networking.k8s.io/v1`) API is available (so far so good). However, the code later on in the file that actually defines the ingress hasn't changed. It needs to be something like

```
      - backend:
          service:
              name: {{ $serviceName }}
              port: 80
```

if using the new API.

**Comment by js-timbirkett** [comment]
THis should probably have been opened on: https://github.com/FairwindsOps/charts/issues and was brought up: https://github.com/FairwindsOps/charts/issues/519

Fixed in: https://github.com/FairwindsOps/charts/pull/545 which got released in chart version 4.0.7, so this can probably be closed.

**Comment by tom-dalton-fanduel** [comment]
Ah that's great, thanks.
